biojava-core/src/main/java/org/biojava/nbio/core/sequence/reference/GenbankReference.java

```diff
@@ -0,0 +1,66 @@
+package org.biojava.nbio.core.sequence.reference;
+
+/**
+ * For Genbank format file only.
+ *
+ * @Author Jim Tang
+ */
+public class GenbankReference extends AbstractReference {
+
+    /**
+     * The authors are a list of Inventors that retrieved from the Reference section.
+     */
+    private String authors;
+
+    /**
+     * The title that retrieved from the Reference section.
+     */
+    private String title;
+
+    /**
+     * The journal usually contains the Publication Number, Publication Date and Assignee
+     */
+    private String journal;
+
+    /**
+     * @return
+     */
+    public String getAuthors() {
```

Comment:
Here we should have \@Override instead of javadocs. The docs are in the interface.